hack/ci/e2e.sh

```diff
@@ -142,13 +142,18 @@ apiVersion: v1
 data:
   Corefile: |
     .:53 {
+        log
         errors
         health
         kubernetes cluster.local in-addr.arpa ip6.arpa {
            pods insecure
+           fallthrough in-addr.arpa ip6.arpa
+           ttl 30
         }
         prometheus :9153
+        forward . /etc/resolv.conf
```

Comment:
hm, so this will forward everything using the host resolv.conf:
https://github.com/coredns/coredns/tree/master/plugin/forward#examples

why is it needed for ipv6?

Reply:
this is for debugging the CI failures, I just took the original CoreDNS configuration and removed the `upstream` field, will try to change only one thing at a time to try to understand why the DNS tests are failing in the CI and working on my host
```
apiVersion: v1
data:
  Corefile: |
    .:53 {
        errors
        health
        kubernetes cluster.local in-addr.arpa ip6.arpa {
           pods insecure
           upstream
           fallthrough in-addr.arpa ip6.arpa
           ttl 30
        }
        prometheus :9153
        forward . /etc/resolv.conf
        cache 30
        loop
        reload
        loadbalance
    }
kind: ConfigMap
metadata:
  creationTimestamp: "2019-07-03T16:30:31Z"
  name: coredns
  namespace: kube-system
```